### DISCUSSION TITLE
Movie poster overwriting

### DISCUSSION THREAD

**Discussion by chain710** [meisnate12/Plex-Meta-Manager#2096] [Q&A] [answered]
Hi

I changed movie's poster in Plex, but it seems that kometa overwrites it every time

For example:
Movie `Kill Bill`

I selected the follwing poster
![image](https://github.com/Kometa-Team/Kometa/assets/486539/3a81c47a-af6a-4194-900d-91b5263eb3c3)

But the next day, it changed to

![image](https://github.com/Kometa-Team/Kometa/assets/486539/8601cb05-b310-40ab-973b-a93b331daf24)

Config:
```
libraries:                           # This is called out once within the config.yml file
  电影:                            # These are names of libraries in your Plex
    remove_overlays: false           # Set this to true to remove all overlays
    collection_files:
      - default: basic               # This is a file within Kometa's defaults folder
      - default: imdb                # This is a file within Kometa's defaults folder
      - default: oscars
      - default: franchise
      - default: universe
      # see the wiki for how to use local files, folders, URLs, or files from git
    overlay_files:
      - default: ribbon              # This is a file within Kometa's defaults folder
      - default: resolution
  电视节目:
    remove_overlays: false           # Set this to true to remove all overlays
    collection_files:
      - default: basic               # This is a file within Kometa's defaults folder
      - default: imdb                # This is a file within Kometa's defaults folder
      - default: anilist
    overlay_files:
      - default: ribbon              # This is a file within Kometa's defaults folder

playlist_files:
  - default: playlist    
plex:                                # Can be individually specified per library as well; REQUIRED for the script to run
  url: http://xxx
  token: xxx
  timeout: 60
  db_cache:
  clean_bundles: false
  empty_trash: false
  optimize: false
  verify_ssl:
tmdb:                                # REQUIRED for the script to run
  apikey: xxx
  language: zh
```

Did I do something wrong?

**Comment by chazlarson** [comment, marked as answer]
This is normal and expected.

If you want to change a poster in Plex after overlays have been applied, you need to remove the "overlay" sharing label from the thing so Kometa knows it should grab that new art.

if you don't do that, then future overlay applications will use the backup art that Kometa saved the first time it did overlays and will stomp your new art.

if you set new art on the thing using the asset directory, you do not need to do this; Kometa will use the asset as the source of truth for which poster you want Star Wars to have.

![image](https://github.com/Kometa-Team/Kometa/assets/3865541/8baa5d03-152f-4354-92ca-39096ce7fde5)

**Comment by chain710** [comment]
Sorry, I didn't notice this part of the document. Thanks a lot for pointing it out for me.
